### ISSUE TITLE
implement blind instant sync for tendermint

### ISSUE THREAD

**Issue by pro-wh** [oasislabs/oasis-core#1445]
Currently having nodes catch up to the current tendermint state when they come on line is a big delay, and we don't need that level of assurance in our current deployment. It is proposed in this issue that we implement a "debug" level option somehow to skip this step. This will unblock some ongoing validation of other functionality that is under development.

## Details

A discussion for a proper solution to the problem of long catching up time is open in #1444.

cc @ryscheng 

## Acceptance Criteria
* a node can join a network in constant time under this debug configuration

**Comment by kostko** [comment]
Superseded by #2880.
